fsharp-backend/src/ApiServer/ApiServer.fs

```diff
@@ -107,7 +107,6 @@ let addRoutes
   addRoute "GET" "/a/{canvasName}/trigger-exception" std R exceptionFn
 
   ocamlCompatibleApi "add_op" RW AddOps.addOp
-  ocamlCompatibleApi "clean_ops" RW AddOps.clean
```

Comment:
What was calling this?

Reply:
Nothing. I used it to covert the opcodes then it didnt have a use.